### PR TITLE
fixed problem with UTF-8 format 

### PR DESCRIPTION
**Pull request by assaabriiii**
i've converted pdf file to text and then to csv, in this case it would solve UTF-8 format errors

### CODE REVIEW

**Review by markstoel — APPROVED** [review]

**Review by assaabriiii — COMMENTED** [review]
reviewed

File: main.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+DIRECTORY="$1"
+
+# Check if the argument is a directory
+if [ ! -d "$DIRECTORY" ]; then
+    echo "Error: $DIRECTORY is not a directory"
+    exit 2
+fi
+
+# Loop over all PDF files in the directory
+for pdf_file in "$DIRECTORY"/*.pdf;
```
Comment (fragtastic) [inline main.sh:19]: Converter uses UTF8 text input, not PDF input.
Will not be doing PDF to TXT conversion internally.

File: test.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import pdftotext
```
Comment (fragtastic) [inline test.py:1]: I've had poor text results using this, if I remember correctly.
Will revisit.

File: cisconverter/__init__.py [hunk 3]
```diff
@@ -61,33 +63,75 @@ def build_blank():
...
         logging.debug(row)
-
+        
     def parse_text(self):
-        with open(self.args.inputFilePath, mode='rt', encoding='utf8') as inFile:
+        os.system(f"pdftotext {self.args.inputFilePath} convertedtotext.txt")
+        
+        with open("convertedtotext.txt", mode='rt', encoding="utf8") as inFile:
             logging.info(f'Parsing {self.args.inputFilePath}')
+            os.system("rm convertedtotext.txt")
```
Comment (fragtastic) [inline cisconverter/__init__.py:81]: Converting PDF to TXT is not appropriate here.

**Comment by fragtastic** [comment]
This overall has nothing to do with UTF-8 errors and is not a solution to issue #23 previously opened.
This is mostly a copy paste of the same changes that another pull request has.

**Pull request closed without merging**